modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -472,16 +472,24 @@ use {{invokerPackage}}\ObjectSerializer;
         $multipart = false;
 
         {{#queryParams}}
+
         // query params
-        {{#collectionFormat}}
+        {{#isExplode}}
+        if (${{paramName}} !== null) {
+            $queryParams['{{baseName}}'] = ${{paramName}};
+        }
+        {{/isExplode}}
```

Comment:
This code doesn't properly support the 'form' style. If we look at other implementations (such as the Javascript one), when explode is true and style equals to 'form', the parameter should be expanded so that each key inside the parameter (which should then be an array) would result in a new query parameter. There is a very nice example of this in the accepted answer for [this thread at stackoverflow](https://stackoverflow.com/questions/49582559/how-to-document-dynamic-query-parameter-names-in-openapi-swagger).
So, to make this work, in my humble opinion this code might be changed to:
```
        if (${{paramName}} !== null) {
        {{#style}}
            if('form' === '{{style}}' && is_array(${{paramName}})) {
                foreach(${{paramName}} as $key => $value) {
                    $queryParams[$key] = $value;
                }
            }
            else {
                $queryParams['{{baseName}}'] = ${{paramName}};
            }
        {{/style}}
        {{^style}}
            $queryParams['{{baseName}}'] = ${{paramName}};
        {{/style}}
        }
```
The only problem here is: PR #4042 is unfortunately incomplete, and due to it, the style property is always empty. I've created a new PR #4640 to fix the issue...

Reply:
EDIT: #4640 is merged now, which means the style property should work fine from now on! :)